src/meta/api/request/geo/layer.ts

```diff
@@ -10,9 +10,9 @@ export type ForestLayerRequest = Request<
   {
     countryIso: CountryIso
     forestSource: ForestSource
-    gteHansenTreeCoverPerc?: string
-    onlyProtected: string
-    opacity: string
+    gteHansenTreeCoverPerc?: any
+    onlyProtected: any
+    opacity: any
```

Comment:
why was this changed to any?

Reply:
When request validation is performed, I convert some query params  to number. I wanted to use directly those sanitized values, but request params cannot be typed with other than string type as I understand. With any I can pass converted params (as numbers) to the controller function, maybe there is a better workaround.

Reply:
@sorja we should leave any. it requires to update ```interface ParsedQs { [key: string]: undefined | string | string[] | ParsedQs | ParsedQs[] | number }``` from the query string namespace. 
or do you think we should try to give it a shoot anyway ?

Reply:
I get a compilation error if typing ReqQuery with other than string or any:
![Screenshot from 2022-11-16 13-47-00](https://user-images.githubusercontent.com/15015525/202242029-ccb29fca-08fc-4ee1-b3ac-13d972dd4b97.png)

"No overload matches this call.
  The last overload gave the following error.
    Argument of type '(req: ForestLayerRequest, res: Response) => Promise<void>' is not assignable to parameter of type 'RequestHandlerParams<ParamsDictionary, any, any, ParsedQs, Record<string, any>>'"

...because this is the interface:
 interface ParsedQs { [key: string]: undefined | string | string[] | ParsedQs | ParsedQs[] }

Reply:
Scope of this PR any is ok